src/test/scala/uk/gov/nationalarchives/tdr/api/service/ValidateFileMetadataServiceSpec.scala

```diff
@@ -424,17 +434,44 @@ class ValidateFileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with
     val mockFileStatusRepository: FileStatusRepository = mock[FileStatusRepository]
     val fixedUUIDSource = new FixedUUIDSource()
     fixedUUIDSource.reset
-    val service = new ValidateFileMetadataService(mockCustomMetadataService, mockFileMetadataRepository, mockFileStatusRepository, fixedTimeSource, fixedUUIDSource)
+    val service = new ValidateFileMetadataService(mockCustomMetadataService, mockFileMetadataRepository, mockFileStatusRepository)
     val mockFields = mockCustomMetadataFields()
 
+    def createExpectedFilestatusRow(fileId: UUID, statusType: String, statusValue: String): Seq[FilestatusRow] = {
+      val mappedMetadataTypeValue = (statusValue, statusType) match {
+        case ("Completed", _)                      => "Completed"
+        case ("Incomplete", "ClosureMetadata")     => "Incomplete"
+        case ("Incomplete", "DescriptiveMetadata") => "NotEntered"
+        case ("Incomplete", _)                     => "Incomplete"
+        case ("NotEntered", _)                     => "NotEntered"
+      }
+      Seq(FilestatusRow(null, fileId, statusType, mappedMetadataTypeValue, null))
```

Comment:
I would still add the status id and timestamp so that the return rows accurately reflect what the DB would return even though you're mocking the  return values, eg: `... FilestatusRow(UUID.random(), id, "statusType", "value", Timestamp.from(FixedTimeSource.now)) ...`